components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -3767,10 +3767,12 @@ private String getUserStoreDomainFromSP() throws IdentityApplicationManagementEx
      * @param userStoreName user store name
      * @return whether scim is enabled or not for the particular user store
      */
-    private boolean isSCIMEnabled(String userStoreName) {
+    private boolean isSCIMEnabled(String userStoreName) throws BadRequestException {
 
         UserStoreManager userStoreManager = carbonUM.getSecondaryUserStoreManager(userStoreName);
-        if (userStoreManager != null) {
+        if (userStoreManager == null) {
+            throw new BadRequestException("Invalid user store name.");
+        }else {
```

Comment:
Formatting issue. Add a space after else

Reply:
fixed in main